pkg/systemd/logs.html

```diff
@@ -67,7 +67,27 @@
 
         <div class="filter text-search">
             <label for="journal-grep" translate>Text</label>
-            <input id="journal-grep" class="form-control" type="text" translate="placeholder" placeholder="Type to filter">
+            <span class="filter-group">
+                <input id="journal-grep" class="form-control" type="text" translate="placeholder" placeholder="Type to filter">
+                <div id="logs-help" class="dropdown">
+                  <a role="link" tabindex="0" id="logs-help-toggle" class="dropdown-toggle dropdown-form-control" data-toggle="dropdown">
+                    <span class="pficon pficon-help"></span>
+                  </a>
+                  <div id="logs-help-menu" class="dropdown-menu">
+                    <span translate>The text entry allows searching the full content of the log entries. It accepts strings such as:</span>
+                    <ul>
+                        <li><span translate>qualifiers</span>, <span translate>E.g</span> 'priority', 'identifier', 'services'</li>
+                        <li><span translate>log fields</span>, <span translate>E.g</span> '_EXE=/usr/bin/python'</li>
+                        <li translate>any free-form string, can be combined using regular expressions</li>
```

Comment:
I'm not sure how regular expressions  can combine things. This means that terms which are not "qualifier:" or "FIELD=" are interepreted as RE, right? So maybe just "a regular expression", and the introduction above covers the "combination" part.